lib/blacklist.js

```diff
@@ -3,11 +3,10 @@ const notifierQueries = require('./notifier/queries')
 
 // Get all blacklist rows from the DB "blacklist" table that were manually inserted by the operator
 const getBlacklist = () => {
-  return db.any(`SELECT * FROM blacklist WHERE created_by_operator = 't'`).then(res =>
+  return db.any(`SELECT * FROM blacklist`).then(res =>
     res.map(item => ({
       cryptoCode: item.crypto_code,
-      address: item.address,
-      createdByOperator: item.created_by_operator
```

Comment:
I'm pretty sure the blacklist type will still have the definition of `createdByOperator`.

Reply:
Correct. Altering the blacklist type should solve this. ATM I'm just removing the createdByOperator without migrating older entries with this field enabled, is this a good approach?